edweiss-app/config/fcm/ForegroundHandler.ts

```diff
@@ -0,0 +1,19 @@
+// import { FCMMessage } from '@/model/fcm';
```

Comment:
Maybe you can delete this file, because there are only comments

Reply:
Yes I am still in the process of communicating with the person that made this file to verify if it is no longer necessary